build.sh

```diff
@@ -128,31 +128,41 @@ if ! [ -f Common/edk2/edksetup.sh ] || ! [ -f ../edk2/edksetup.sh ]
 then	set -e
 	echo "Updating submodules"
 	if "${CHINESE}"
-	then	git submodule set-url edk2 https://hub.fastgit.xyz/tianocore/edk2.git
-		git submodule set-url edk2-platforms https://hub.fastgit.xyz/tianocore/edk2-platforms.git
-		git submodule set-url GPLDrivers/Library/simple-init https://hub.fastgit.xyz/BigfootACA/simple-init.git
-		git submodule init;git submodule update --depth 1
+	then
+		vim -u NONE -N .gitmodules -c "%s/github.com/hub.nuaa.cf/g" -c ":wq"
+		git submodule init;git submodule update
 		pushd Common/edk2
-
-		git submodule set-url ArmPkg/Library/ArmSoftFloatLib/berkeley-softfloat-3   https://hub.fastgit.xyz/ucb-bar/berkeley-softfloat-3.git
-		git submodule set-url CryptoPkg/Library/OpensslLib/openssl                  https://hub.fastgit.xyz/openssl/openssl.git
-		git submodule set-url BaseTools/Source/C/BrotliCompress/brotli              https://hub.fastgit.xyz/google/brotli.git
-		git submodule set-url UnitTestFrameworkPkg/Library/CmockaLib/cmocka         https://git.cryptomilk.org/projects/cmocka.git
-		git submodule set-url ArmPkg/Library/ArmSoftFloatLib/berkeley-softfloat-3   https://hub.fastgit.xyz/ucb-bar/berkeley-softfloat-3.git
-		git submodule set-url MdeModulePkg/Library/BrotliCustomDecompressLib/brotli https://hub.fastgit.xyz/google/brotli.git
-		git submodule set-url MdeModulePkg/Universal/RegularExpressionDxe/oniguruma https://hub.fastgit.xyz/kkos/oniguruma.git
+		vim -u NONE -N .gitmodules -c "%s/github.com/hub.nuaa.cf/g" -c ":wq"
```

Comment:
*  I don't want to use additional tools for modifications
*  I think git submodule set url is a better solution
*  Please avoid directly modifying '.getmodules' (vim)